composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "tawk/tawk-wordpress",
+    "description": "Wordpress plugin for tawk.to",
+    "type": "project",
+    "license": "OSL-3",
+    "version": "0.6.0",
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.6",
+        "wp-coding-standards/wpcs": "^2.3",
+        "phpcompatibility/php-compatibility": "^9.3",
+        "woocommerce/woocommerce-sniffs": "^0.1.1"
+    },
+    "scripts": {
+        "post-install-cmd": "phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs,vendor/phpcompatibility/php-compatibility",
+        "lint": "phpcs -p -s -v --runtime-set ignore_warnings_on_exit true tawkto",
+        "lint:fix": "phpcbf -p -s -v tawkto; err=$?; if [ $err -eq 1 ]; then exit 0; else exit $err; fi;"
```

Comment:
Made a custom handler for exit code for the command `phpcbf` because, according to [this comment](https://github.com/squizlabs/PHP_CodeSniffer/issues/1818#issuecomment-647330553) in one of the issues reported to PHPCS, here are the current exit codes for `phpcbf`:
- Exit code 0 is used to indicate that no errors were found
- Exit code 1 is used to indicate that errors were found but can be fixed
- Exit code 2 is used to indicate that errors were found and some can be fixed

So I made a handler for exit code 1 to have an exit code of 0 just so `lint-staged` won't fail.